pom.xml

```diff
@@ -0,0 +1,362 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.cdap.plugin</groupId>
+    <artifactId>zuora</artifactId>
+    <version>1.3.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <description>Zuora Plugin</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <cdap.version>6.1.0-SNAPSHOT</cdap.version>
+        <hadoop.version>2.8.0</hadoop.version>
+        <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
+        <gson.version>2.2.4</gson.version>
+        <common.logging.version>1.2</common.logging.version>
+        <junit.version>4.11</junit.version>
+        <mockito.version>1.10.19</mockito.version>
+        <spark.version>2.1.3</spark.version>
```

Comment:
seems like this and some other are not used

Reply:
they would be used with added integration tests